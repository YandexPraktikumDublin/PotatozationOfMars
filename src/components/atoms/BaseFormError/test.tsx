import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { BaseFormError } from '.'

describe('<BaseFormError />', () => {
  const children = 'Test children'

  it('should renders correct <BaseFormError />', () => {
    const wrapper = shallow(<BaseFormError>{children}</BaseFormError>)

    expect(wrapper.contains(children)).toBeTruthy()

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
