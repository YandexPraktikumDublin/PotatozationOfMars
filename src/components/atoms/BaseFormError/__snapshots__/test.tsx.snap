// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BaseFormError /> should renders correct <BaseFormError /> 1`] = `
<div
  className="text-sm text-danger text-center mt-1"
>
  Test children
</div>
`;
